### ISSUE TITLE
Female ridership tracking

### ISSUE THREAD

**Issue by bradwilliford** [kevinburke/gobike#36]
For Ford GoBike, currently female ridership is approx 26%.

As SF installs safer bike lanes and bike share spreads throughout the city, we're hopeful to improve this number and get more women biking.

Is it possible to had a graph that shows the breakdown of gender (Male / Female / Other)?

**Comment by kevinburke** [comment]
Maybe the right way to do this would be to have multiple lines on the "Trips per week" chart - one line for male, one female, one other, one total.

**Comment by bradwilliford** [comment]
That would work. Would it be possible to hover over it to see the percentage? A key metric we're hoping to hit is 30% female ridership.

**Comment by kevinburke** [comment]
Yep! FYI all of the plotting source code is here - would need to customize the legend for that chart & add the gender breakouts, but should be possible. https://github.com/kevinburke/gobike/blob/master/templates/city.html#L336

Scanning the values in one of the tripdata.csv files I'm worried about mis-classification, maybe it has gotten better but there are a lot of empty values meaning people aren't identifying.
